manifests/generated/kv-resource.yaml

```diff
@@ -890,10 +890,15 @@ spec:
                           AutoattachSerialConsole is disabled.
                         type: object
                     type: object
+                  vmStateAccessMode:
+                    description: VMStateAccessMode is the name of the persistent volume
```

Comment:
We probably don't want to do this. CDI already tracks access modes for various storage classes, so we could just reference that.